### PR TITLE
a  small cleanup

### PR DESCRIPTION
**Pull request by kingarrrt**
Minor fixes.

### CODE REVIEW

**Comment by kingarrrt** [comment]
Sorry for the push noise. I'll send along the other stuff once this is agreed.

**Comment by bbenne10** [comment]
This looks great. Thank you for your effort. I'll let @Mic92 weigh in, but I'm happy to merge after that.

**Review by bbenne10 — APPROVED** [review]
This all looks good from my pov.

File: direnvrc [hunk 1]
```diff
@@ -27,26 +27,28 @@ _nix_direnv_fatal() {
...
-    if [[ -z "${NIX_BIN_PREFIX}" ]]; then
-      _nix_direnv_fatal "command not found: nix-shell."
-    fi
-    NIX_BIN_PREFIX="${NIX_BIN_PREFIX%/*}/"
+  if ! has direnv_version || ! direnv_version "$REQUIRED_DIRENV_VERSION" 2>/dev/null; then
+    _nix_direnv_fatal "base direnv version is older than the required v$REQUIRED_DIRENV_VERSION."
   fi
 
+  if [[ -n ${NIX_BIN_PREFIX:-} ]]; then
+    PATH_add "$NIX_BIN_PREFIX"
```
Comment (Mic92) [inline direnvrc:44]: maybe we can append NIX_BIN_PREFIX later on... so that if the user has nix in PATH it will be used.

**Pull request merged**